### PR TITLE
Client product detail

### PR DESCRIPTION
**Pull request by hduong1207uet**
## Related Tickets:
- [#35066](https://edu-redmine.sun-asterisk.vn/issues/35066)

## WHAT this PR does?
Client page - View product detail

## Notes Impacted Areas:
- N/A.

## Attachments:
- Guitar Shop (2)

### CODE REVIEW

**Comment by hduong1207uet** [comment]
ready

File: app/Http/Controllers/HomeController.php [hunk 1]
```diff
@@ -116,4 +116,17 @@ public function removeFromCart($id)
         }
         abort(404);
```
Comment (congnx-2423) [inline app/Http/Controllers/HomeController.php]:
```suggestion
        return abort(Response::HTTP_NOT_FOUND);
```
tranh hardcode nhe

File: resources/js/app.js [hunk 2]
```diff
@@ -73,4 +73,24 @@ $(document).ready(function () {
...
+        let quantity = parseInt($('#quantity').val());
+            $('#quantity').val(quantity + 1);
+    });
+
+    $('.quantity-left-minus').click(function (e) {
+        // Stop acting like a button
+        e.preventDefault();
+        // Get the field name
+        let quantity = parseInt($('#quantity').val());
+            if (quantity>0) {
```
Comment (congnx-2423) [inline resources/js/app.js]:
```suggestion
            if (quantity > 0) {
```

**Comment by hduong1207uet** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**